src/scripts/modules/media/body/body.coffee

```diff
@@ -152,6 +152,31 @@ define (require) ->
             # NOTE: `.toggleClass()` explicitly requires a `false` (not falsy) 2nd argument
             $el.toggleClass('ui-has-child-title', $title.length > 0)
 
+          # Create subtitles in each chapter with corresponding ids as their href
```

Comment:
I think that the code should not error when a user goes to an individual page. For example, https://cnx.org/contents/bG-_rWXy@5/Introduction (there is no left hand sidebar)